src/comps/Form/CreateForm.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import CreateButton from "./CreateButton.js";
+import "./form.css";
+// props: typing, handleChange, handleSubmit, handleClear
+export default props => {
+  function preventSubmit(e) {
+    e.preventDefault();
+    props.handleSubmit();
+  }
+
+  return (
+    <form onSubmit={preventSubmit} className="form">
+      <input
+        type="text"
+        placeholder="...to do"
+        value={props.typing}
+        onChange={props.handleChange.bind(this, "typing")}
```

Comment:
good job. I know making forms in React can be a pain (especially when a LOT of people forget to add the .bind like you did on line 17 lol). Well done